opendevin/requirement.txt

```diff
@@ -0,0 +1,9 @@
+docker
```

Comment:
I think these are already in `/requirements.txt`, no?

Reply:
Oh, sorry, just find the we already have a `requirement.txt` in root folder. Maybe I forget to update the codebase when dev. Will update this PR.

Reply:
Removed.